server/main.lua

```diff
@@ -156,11 +203,34 @@ lib.callback.register('vehiclekeys:server:RemoveKey', function(source, netId, ta
     -- This callback is not yet implemented
 end)
 
---- Toggles the door state of the vehicle between open and closed.
+---Toggles the door state of the vehicle between open and closed.
 ---@param source number ID of the player
 ---@param netId number The network ID of the entity
 ---@return number | nil -- Returns the current Door State
 lib.callback.register('vehiclekeys:server:ToggleDoorState', function(source, netId)
     if not source or not netId then return end
     -- This callback is not yet implemented
 end)
+
+---Returns if the vehicle is owned by a player or not
+---@param plate string
+---@return boolean
+lib.callback.register('vehiclekeys:server:IsPlayerOwned', function(_, plate)
+    return not not vehicleList[plate]
+end)
+
+-----------------------
+----   Threads     ----
+-----------------------
+
+CreateThread(function()
+    local vehicles = MySQL.query.await('SELECT * FROM player_vehicles')
+    for i = 1, #vehicles do
+        local data = vehicles[i]
+        vehicleList[data.plate] = {}
+
+        if data.citizenid then
+            vehicleList[data.plate][data.citizenid] = true
+        end
+    end
+end)
```

Comment:
Please add newlines at end of files

Reply:
no

Reply:
Another case of team alignment needed

Reply:
lack of eol can make it harder to compare changes
![obraz](https://github.com/Qbox-project/qbx_vehiclekeys/assets/36608116/4cd78cab-416f-4d6f-a21f-d7ffb02147d1)

Reply:
> Another case of team alignment needed

Why would you even add newlines under files? It seems incredibly pointless to create an empty space where nothing continues.

Reply:
> lack of eol can make it harder to compare changes ![obraz](https://private-user-images.githubusercontent.com/36608116/329815949-4cd78cab-416f-4d6f-a21f-d7ffb02147d1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU0Njc2NDgsIm5iZiI6MTcxNTQ2NzM0OCwicGF0aCI6Ii8zNjYwODExNi8zMjk4MTU5NDktNGNkNzhjYWItNDE2Zi00ZDZmLWEyMWYtZDdmZmIwMjE0N2QxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTExVDIyNDIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUzYzQ0OTIxZGRkMDA0ODI2MWExMWU1YjYwODY2Y2Y0NTY1YzZiNjJhOGE2NDU0OTNiYTI1MWJmNjUwNmZmYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tMsmJl4GWyVS0qPdEayT9dciaeoKo3KhbATSWdaxR7A)

I did not know that, thanks for that info.

Reply:
POSIX standard. The EOL at the end of a file ensures that all lines, including the last one, are treated consistently by various tools and utilities, while the EOF is a control character used to signify the end of data.